CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,7 @@
+# Contributor Code of Conduct
```

Comment:
@Bhupesh-V do we need another project-specific `CODE_OF_CONDUCT.md`? The [organization-wide `CODE_OF_CONDUCT.md`](https://github.com/developersIndia/.github/blob/main/CODE_OF_CONDUCT.md) will be used everywhere regardless.

Reply:
No, we don't need another COC, the organisation one is accessible in this repo as well. Let's remove that for now